### PR TITLE
[YUNIKORN-1761] Release notes for 1.3.0

### PR DESCRIPTION
**Pull request by craigcondit**
### What is this PR for?
Release notes for YuniKorn 1.3.0.

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [x] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1761

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: src/pages/release-announce/1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+-->
+
+# Release Announcement v1.3.0
+We are pleased to announce that the Apache YuniKorn community has voted to
+release 1.3.0. Apache YuniKorn is a standalone resource scheduler, designed
+for managing and scheduling Big Data workloads on container orchestration
+frameworks like Kubernetes for on-prem and on-cloud use cases.
+
+## Overview
+The Apache YuniKorn community has resolved 164 [JIRAs](https://issues.apache.org/jira/issues/?filter=12352777)
```
Comment (wilfred-s) [inline src/pages/release-announce/1.3.0.md:32]: It needs to be this filter:
https://issues.apache.org/jira/issues/?filter=12352568
This one stays and is just for this release the next release filter will be updated as soon as we release causing a breakage in the page

File: src/pages/release-announce/1.3.0.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+### Kubernetes version support
+YuniKorn 1.3.0 supports running on Kubernetes clusters from version 1.21
+through 1.26. Support for running on Kubernetes 1.27 is **not** supported,
+but is planned for YuniKorn 1.4.0.
+
+**NOTE**: YuniKorn 1.3.0 is the last release planned which will support
+Kubernetes 1.21 through 1.23; YuniKorn 1.4.0 is expected to require Kubernetes
+1.24 or later.
+
```
Comment (wilfred-s) [inline src/pages/release-announce/1.3.0.md:49]: Add a jira reference:
```
Please see [[YUNIKORN-1699](https://issues.apache.org/jira/browse/YUNIKORN-1699) for details.
```

File: src/pages/release-announce/1.3.0.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+### Preemption support
+YuniKorn now has comprehensive support for preemption, including customizable
+policies per queue. Please see the the Preemption [design documentation](https://yunikorn.apache.org/docs/next/design/preemption)
+for more details.
```
Comment (wilfred-s) [inline src/pages/release-announce/1.3.0.md:53]: Add a jira link:
```
Jira: [YUNIKORN-1461](https://issues.apache.org/jira/browse/YUNIKORN-1461)
```

File: src/pages/release-announce/1.3.0.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+### Better support for BestEffort pods
+YuniKorn now tracks the number of pods scheduled as part of queue quota limits.
+This allows limiting parallelism within a queue, but also provides better
+support for running Kubernetes pods with the `PodQOSBestEffort` scheduling class
+(i.e. those with no explicit resources requested).
```
Comment (wilfred-s) [inline src/pages/release-announce/1.3.0.md:59]: Add a jira reference:
```
Please see [YUNIKORN-1632](https://issues.apache.org/jira/browse/YUNIKORN-1632) for details.
```

File: src/pages/release-announce/1.3.0.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+### Lighter and more secure images
+YuniKorn 1.3.0 has moved to using scratch base images for all containers. This
+results in a smaller footprint as well as improved security as YuniKorn no
+longer ships any third-party software in our container images.
```
Comment (wilfred-s) [inline src/pages/release-announce/1.3.0.md:64]: Add the drop of nginx and jira refs:
```
A prerequisite for using scratch images was the move from nginx to a go based web server for the web application.
Please see [YUNIKORN-1672](https://issues.apache.org/jira/browse/YUNIKORN-1672) and [YUNIKORN-1535](https://issues.apache.org/jira/browse/YUNIKORN-1535) for details.
```

File: src/pages/release-announce/1.3.0.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+### Dynamic namespace support in admission controller
+The YuniKorn admission controller now suports per-namespace annotations to
+control YuniKorn-specific processing. Previously, only a static list of
+regular expressions (still supported) could be used to select namespaces.
```
Comment (wilfred-s) [inline src/pages/release-announce/1.3.0.md:69]: jira ref:
```
Please see [YUNIKORN-1647](https://issues.apache.org/jira/browse/YUNIKORN-1647) for details.
```

File: src/pages/release-announce/1.3.0.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+### CPU and memory optimization
+YuniKorn 1.3.0 contains several optimizations for CPU and memory usage, making
+this release the lightest and fastest yet.
```
Comment (wilfred-s) [inline src/pages/release-announce/1.3.0.md:73]: add some examples:
```
Some examples:
* [YUNIKORN-1719](https://issues.apache.org/jira/browse/YUNIKORN-1719) Improve the performance of Application.sortRequests()
* [YUNIKORN-1720](https://issues.apache.org/jira/browse/YUNIKORN-1720) Improve the performance of node.preAllocateCheck()
* [YUNIKORN-1722](https://issues.apache.org/jira/browse/YUNIKORN-1722) excessive resource clone calls in calculateNodesResourceUsage
* [YUNIKORN-1746](https://issues.apache.org/jira/browse/YUNIKORN-1746) Improve the performance of nodeInfoListerImpl.List()
```

File: src/pages/release-announce/1.3.0.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+## Community
+The Apache YuniKorn community is pleased to welcome new PMC member TODO and committer TODO.
```
Comment (wilfred-s) [inline src/pages/release-announce/1.3.0.md:76]: PMC and new committer are announced now

**Review by wilfred-s — CHANGES_REQUESTED** [review]
mostly jira ref updates

**Comment by craigcondit** [comment]
Addressed review feedback. Date is tentatively this Friday, will update with final date and remove Draft status once release is finished.

**Review by wilfred-s — APPROVED** [review]
LGTM: @0yukali0 
Only thing left is to update the release date just before commit if we miss the date

**Review by wilfred-s — APPROVED** [review]
LGTM: @0yukali0 
Only thing left is to update the release date just before commit if we miss the date

**Comment by 0yukali0** [comment]
Understood.

**Pull request closed without merging**